fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/patch/Main.java

```diff
@@ -69,9 +68,9 @@ public class Main {
     // Arguments requesting we drop the objects from the schema
     private boolean dropOldConstraints = true;
 
-    // The database type being populated (default: Db2)
-    private DbType dbType = DbType.DB2;
-    private IDatabaseTranslator translator = new Db2Translator();
+    // The database type being populated (default: PostgreSQL)
```

Comment:
I can't even remember what this alternative Main class was used for now ...wonder if we could drop or deprecate for 5.0

Reply:
Yeah it was to drop some constraints related to composites. We can get rid of this now, especially as we're asserting that upgrades should be based on 4.11.1 (in which case this would already have been applied, if required).